enumerable_methods.rb

```diff
@@ -0,0 +1,65 @@
+module Enumerable
+  def my_each
+    return unless block_given?
+
+    for i in self
+      yield i
+    end
+  end
```

Comment:
- [ ] enumerables #my_each returns an Enumerator if no block is given
     Failure/Error: expect(array.my_each).to be_an(Enumerator)
- [ ]

Reply:
@mimipeshy Hey I do not quite understand what the issue is here, can you please explain the error further?

Reply:
when no block is given, it returns an enumerator, i.e return to_enum(:my_each) unless block_given?